consensus/protocol.go

```diff
@@ -26,7 +26,7 @@ var (
 		Versions: []uint{Istanbul100, Istanbul99, Istanbul64},
 		// istanbul/100 has to have 18 message to be backwards compatible although at the p2p layer it only has
 		// 1 message with msg.Code 17
-		Lengths: map[uint]uint64{Istanbul100: 18, Istanbul99: 18, Istanbul64: 18},
+		Lengths: map[uint]uint64{Istanbul100: 22, Istanbul99: 18, Istanbul64: 18},
```

Comment:
Is the requirement of length `18` for `istanbul/100` backward compatibility mentioned in the comment no longer true?  If so the comment should be updated.

Reply:
Are there any potential impacts for users by making this change to the `istanbul/100` protocol?

Reply:
I will be updating the comment. I don't see any impact of updating number of messages to 22 during upgrade

Reply:
I updated the comment to reflect that `istanbul/100` need to have 22 messages. In case of legacy IBFT, only 18 are used and rest 4 messages are used when QBFT consensus is enabled